build.sbt

```diff
@@ -1,139 +1,139 @@
-  .dependsOn(coreJVM)
-  .enablePlugins(JmhPlugin)
-  .settings(
-    libraryDependencies ++= Seq(
-      "org.sangria-graphql" %% "sangria"       % "1.4.2",
-      "org.sangria-graphql" %% "sangria-circe" % "1.2.1"
-    )
-  )
-
-val commonSettings = Def.settings(
-  scalacOptions ++= Seq(
-    "-deprecation",
-    "-encoding",
-    "UTF-8",
-    "-explaintypes",
-    "-Yrangepos",
-    "-feature",
-    "-language:higherKinds",
-    "-language:existentials",
-    "-unchecked",
-    "-Xlint:_,-type-parameter-shadow",
-    "-Ywarn-numeric-widen",
-    "-Ywarn-unused:patvars,-implicits",
-    "-Ywarn-value-discard"
-  ) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, 12)) =>
-      Seq(
-        "-Xsource:2.13",
-        "-Yno-adapted-args",
-        "-Ypartial-unification",
-        "-Ywarn-extra-implicit",
-        "-Ywarn-inaccessible",
-        "-Ywarn-infer-any",
-        "-Ywarn-nullary-override",
-        "-Ywarn-nullary-unit",
-        "-opt-inline-from:<source>",
-        "-opt-warnings",
-        "-opt:l:inline"
-      )
-    case _ => Nil
-  })
-)
+import sbtcrossproject.CrossPlugin.autoImport.{ crossProject, CrossType }
+
+val mainScala = "2.12.10"
+val allScala  = Seq("2.13.1", mainScala)
+
+inThisBuild(
+  List(
+    organization := "com.github.ghostdogpr",
+    homepage := Some(url("https://github.com/ghostdogpr/caliban")),
+    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    scalaVersion := mainScala,
+    parallelExecution in Test := false,
+    pgpPassphrase := sys.env.get("PGP_PASSWORD").map(_.toArray),
+    pgpPublicRing := file("/tmp/public.asc"),
+    pgpSecretRing := file("/tmp/secret.asc"),
+    scmInfo := Some(
+      ScmInfo(url("https://github.com/ghostdogpr/caliban/"), "scm:git:git@github.com:ghostdogpr/caliban.git")
+    ),
+    developers := List(
+      Developer(
+        "ghostdogpr",
+        "Pierre Ricadat",
+        "ghostdogpr@gmail.com",
+        url("https://github.com/ghostdogpr")
+      )
+    ),
+    crossScalaVersions := allScala
+  )
+)
+
+ThisBuild / publishTo := sonatypePublishToBundle.value
+
+name := "caliban"
+addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
+addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
+
+lazy val root = project
+  .in(file("."))
+  .enablePlugins(ScalaJSPlugin)
+  .settings(skip in publish := true)
+  .settings(historyPath := None)
+  .aggregate(coreJVM, coreJS, http4s)
+
+lazy val core = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("core"))
+  .settings(name := "caliban")
+  .settings(commonSettings)
+  .settings(
+    testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")),
+    libraryDependencies ++= Seq(
+      "com.lihaoyi"    %%% "fastparse"    % "2.1.3",
+      "com.propensive" %%% "magnolia"     % "0.12.0",
+      "dev.zio"        %%% "zio"          % "1.0.0-RC15",
```

Comment:
RC15 was published yesterday, but there wasn't a release of interop-cats yet, so the Example project doesn't compile. Can you move it to RC14 instead? We can bump to RC15 once there is a corresponding interop-cats.

Btw as soon as ZIO hits 1.0, it will be binary compatible so this won't happen. Hopefully next month!